qiskit/transpiler/passes/optimization/crosstalk_adaptive_schedule.py

```diff
@@ -719,8 +719,8 @@ def run(self, dag):
         """
         self.dag = dag
 
+        self.qubit_indices = {bit: dag.find_bit(bit).index for bit in dag.qubits}
```

Comment:
```suggestion
        self.qubit_indices = {bit: idx for idx, bit in enumerate(dag.qubits)}
```
This should be changed back to the original.